R/wrappers.R

```diff
@@ -5,7 +5,7 @@
 coalesce    = function(...) .Call(Ccoalesce, list(...), FALSE)
 setcoalesce = function(...) .Call(Ccoalesce, list(...), TRUE)
 
-fifelse = function(test, yes, no) .Call(CfifelseR, test, yes, no)
+fifelse = function(test, yes, no, na=NULL) .Call(CfifelseR, test, yes, no, na)
```

Comment:
NA I think complicates things in terms of checking whether it has the same type as yes/no

Reply:
agree it complicates but now the `args(fifelse)` returns misleading feedback to user. We could eventually pass extra arg to C `missing(na)` which will make it easier.

Reply:
yes exactly it does, that is why I used `NULL`

Reply:
Best if we can not use default value and just check `missing(na)` from C. And if missing then use NA of appropriate type, otherwise confirm types are matching.

Reply:
there is [R_MissingArg](https://github.com/wch/r-source/blob/09b5b3f820648c20b088ba922f617848c0c00ab9/src/include/Rinternals.h#L878)